rts/lra/lra-test/basic/src/test/java/io/narayana/lra/arquillian/NonRootLRAParticipantIT.java

```diff
@@ -77,4 +81,32 @@ public void testNonRootLRAParticipantEnlist() {
         Assert.assertEquals("Non root JAX-RS participant should have been enlisted and invoked",
             1, counterValue);
     }
+
+    /*
+     * Test that a participant can join an LRA using the Narayana specific client API
+     * Test that the LRA has the correct status of Closing if a participant
+     * returns 202 Accepted when asked to complete.
+     * @throws URISyntaxException if arquillian URL is invalid
+     */
+    @Test
+    public void testFinishLRA() throws URISyntaxException {
+        NarayanaLRAClient client = new NarayanaLRAClient(); // the narayana client API for using LRAs
+        URI lraId = client.startLRA("testFinishLRA");
+        URI compensateURI = new URI(baseURL.toExternalForm() + "/participant/compensate");
+        URI completeURI = new URI(baseURL.toExternalForm() + "/participant/complete");
+
+        // enlist the participant with these compensate and complete URIs into the LRA
+        client.joinLRA(lraId, 0L, compensateURI, completeURI, null, null, null, null, null);
```

Comment:
not relevant to this PR, but this reminds me that maybe we could introduce builder pattern to the joinLRA invocation

Reply:
Noted